docs/docs/extensions/virtuallistview.md

```diff
@@ -142,6 +142,14 @@ interface VirtualListViewItemInfo {
 // Should the list animate additions, removals and moves within the list?
 animateChanges?: boolean;
 
+// Cell recycling is an optimization to re-use similar VLV cells & decrease the cost
+// of render operations. However, in some cases, disabling cell recycling can be
+// benificial, recycled cells continue their regular react lifecycle even when not
+// visible, which can lead to excessive background re-rendering in some cases. When
+//  combining VLV with react libraries (like ReSub) that have subscriptions managed
+// by components can cause this behaviour to manifest.
+disableCellRecycling?: boolean;
```

Comment:
I don't think we should add this. It's unnecessary. If you don't want to use cell recycling, simply don't provide an "itemTemplate" field.

Reply:
Template is a required field on VLV

Reply:
Hmm, it wasn't previously. We could switch it back to optional, as it was intended. Or you can just specify an empty string for the template, which will also disable recycling for those items.

Reply:
Looks like since it's OSS introduction it's been required.  I'll make it optional but will leave a comment in documentation similar to the above comment